source/developer/bot-accounts.rst

```diff
@@ -0,0 +1,186 @@
+.. _bot-accounts:
+
+Bot Accounts
+========================================
+
+Use Bot Accounts to integrate with Mattermost through `plugins <https://developers.mattermost.com/extend/plugins/>`_ or the `Mattermost RESTful API <https://api.mattermost.com>`_. Bot accounts access the RESTful API on behalf of a bot through the use of the :doc:`personal access tokens feature <personal-access-tokens>`.
+
+// TODO screenshot. Waiting for bug fix where channel header is missing from bot DMs
+
+Bot accounts are just like user accounts, except they:
+
+  - Cannot be logged into, and bypass all restrictions related to authentication, such as multi-factor authentication (MFA) and custom terms of service.
+  - Cannot be used to create other bot accounts.
+  - Do not count as a registered user and therefore do not count towards the total users for an Enterprise Edition license.
+
+Given bot accounts cannot be used to log in, they enable developers to create integrations that don’t require an email/password login or any other authentication method for their bots. This removes unnecessary attack surface caused by creating and in some cases circumventing authentication methods to create a bot account.
+
+Additional benefits include:
+
+  - Bot accounts can post to any channel in the system, whether it’s a private team, private channel or a Direct Message channel. Access for bot accounts is controlled by System Administrators.
```

Comment:
No they can't. They have to be added like any other user. It's just that plugins use their super override when using them so it looks like they can.

They can be assigned system admin permissions though.

Reply:
Propose `Bot accounts can be enabled to post to any channel in the system by System Administrators, including a private team, private channel or a Direct Message channel.`

Does this make it more clear that bots "can" post to any channels if they are configured to do so by the SysAdmin

Reply:
Looks good.